src/main/java/net/sf/jabref/gui/mergeentries/MergeEntries.java

```diff
@@ -35,77 +33,75 @@
 import net.sf.jabref.JabRefPreferences;
 import net.sf.jabref.MetaData;
 import net.sf.jabref.logic.l10n.Localization;
-import net.sf.jabref.util.Util;
 import net.sf.jabref.logic.util.strings.StringUtil;
-import net.sf.jabref.gui.undo.NamedCompound;
-import net.sf.jabref.gui.undo.UndoableInsertEntry;
-import net.sf.jabref.gui.undo.UndoableRemoveEntry;
+import net.sf.jabref.gui.PreviewPanel;
 
-import com.jgoodies.forms.builder.ButtonBarBuilder;
 import com.jgoodies.forms.layout.CellConstraints;
 import com.jgoodies.forms.layout.FormLayout;
 import com.jgoodies.forms.layout.RowSpec;
 import com.jgoodies.forms.layout.ColumnSpec;
 
-// created by : ?
-//
-// modified : r.nagel 2.09.2004
-//            - insert close button
-public class MergeEntriesDialog extends JDialog {
+/**
+ * @author Oscar
+ *
+ * Class for dealing with merging entries
+ */
+
+public class MergeEntries {
 
-    // private String [] preferedOrder = {"author", "title", "journal", "booktitle", "volume", "number", "pages", "year", "month"};
     private final String[] columnHeadings = {Localization.lang("Field"),
             Localization.lang("First entry"),
-            Localization.lang("Use 1st"),
+            "\u2190 " + Localization.lang("Use"),
```

Comment:
Is 1st and 2nd really language independent? And can you please use constants instead of just the unicode as for me I do not know all unicode signs by hard :)